mesa/examples/basic/conways_game_of_life/app.py

```diff
@@ -54,15 +55,11 @@ def post_process(ax):
 # Create initial model instance
 model1 = ConwaysGameOfLife()
 
-# Create visualization elements. The visualization elements are solara components
-# that receive the model instance as a "prop" and display it in a certain way.
-# Under the hood these are just classes that receive the model instance.
-# You can also author your own visualization elements, which can also be functions
-# that receive the model instance and return a valid solara component.
-SpaceGraph = make_space_component(
-    agent_portrayal, post_process=post_process, draw_grid=False
-)
-
+renderer = SpaceRenderer(model1, backend="matplotlib")
+# In this case the renderer only draws the agents because we only want to observe
+# the state of the agents, not the structure of the grid.
+renderer.draw_agents(agent_portrayal=agent_portrayal)
```

Comment:
only is repeated... should read 

In this case the renderer only draws the agents because we just want to observe